shims/shims_for_IE.js

```diff
@@ -18,9 +18,9 @@ require('html5-history-api');  // HTML5 History: IE9.
 if (!Object.hasOwnProperty('name')) {
   Object.defineProperty(Function.prototype, 'name', {
     get: function getName() {
-      const matches = this.toString()
+      var matches = this.toString()
```

Comment:
I would prefer to keep this in ES6 (or at least the parts of ES6 that are supported by node 4, the current long-term-support version of NodeJS).

Is this change there to satisfy ESLint?